Tatsi/Config/TatsiColors.swift

```diff
@@ -21,6 +21,9 @@ public protocol TatsiColors {
 
     /// The color for secondary labels like descriptions.
     var secondaryLabel: UIColor { get }
+    
+    /// The color for the checkmark when selecting an image
+    var checkMark: UIColor { get }
```

Comment:
It would be better to make `checkMark` an optional. See further down for an explanation :)

```suggestion
    var checkMark: UIColor? { get }
```

Reply:
I made the checkMark option, but unfortunately it still needs to be provided in the protocol